UI/src/view/point_asset/massTransitMarker.js

```diff
@@ -146,23 +154,81 @@
       return cacheImage(cachedImageKey, image, canvas.width, canvas.height );
     };
 
+    function extractServicePointsAuxiliarsValues() {
+      var resultValues = {palvelu: undefined, tarkenne: undefined};
+
+      if (data.propertyData !== undefined) {
+        resultValues.palvelu = data.propertyData.find(function (prop) { if (prop.publicId == "palvelu") return prop; });
+        resultValues.tarkenne = data.propertyData.find(function (prop) { if (prop.publicId == "tarkenne") return prop; });
```

Comment:
change data.propertyData.find.... to _.find(data.propertyData, function(prop) ..... )

Reply:
Done